src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -50,12 +50,12 @@ ansys-saf-glow = {version = "0.4.0", source = "solutions-private-pypi"}
 ansys-saf-pim-light-server = {version = "^0.2.0", source = "solutions-private-pypi"}
 ansys-saf-portal = {version = "0.5.dev0", source = "solutions-private-pypi"}
 ansys-solutions-optislang-frontend-components = {version = "0.1.dev11", source = "solutions-private-pypi"}
+ansys-web-components-dash = {version = "^0.0.dev2", source = "solutions-private-pypi"}
```

Comment:
I guess you should remove ansys-dash-treeview dependency?

Reply:
Yes done